### PR TITLE
Piatek

### PR DESCRIPTION
**Pull request by ewelina-jazurek**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -37,3 +37,6 @@ Your Pages site will use the layout and styles from the Jekyll theme you have se
 ### Support or Contact
 
 Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.
+dzis jest piatek
+zosia
```
Comment (grzegorz-jazurek) [inline index.md]: this is out of context, please remove it

**Review by grzegorz-jazurek — CHANGES_REQUESTED** [review]
please update code

File: index.md (hunk 1 above)
Comment (ewelina-jazurek) [reply index.md]: ok, zosia removed

**Review by grzegorz-jazurek — APPROVED** [review]
lgtm

**Pull request merged**